camerakit/src/main/java/com/flurgle/camerakit/CameraView.java

```diff
@@ -480,6 +493,16 @@ public void onCameraClosed() {
         @Override
         public void onPictureTaken(byte[] jpeg) {
             super.onPictureTaken(jpeg);
+
+            // Handle cameras that don't give us the correctly rotated/mirrored image, but instead just set the corresponding EXIF data.
+            // Exif data is lost when we do a BitmapFactory.decodeByteArray, so need to correct image here.
+            if(ExifUtil.getExifOrientation(jpeg) != ExifInterface.ORIENTATION_NORMAL || mFacing == FACING_FRONT){
+                Bitmap bitmap = ExifUtil.decodeBitmapWithRotation(jpeg, mFacing == FACING_FRONT);
+                ByteArrayOutputStream stream = new ByteArrayOutputStream();
+                bitmap.compress(Bitmap.CompressFormat.PNG, 100, stream);
```

Comment:
Is compressing using `CompressFormat.PNG` intentional? I don't know much about this lib, but I would think that `mJpegQuality` should be respected when compressing the bitmap here. 

```
bitmap.compress(Bitmap.CompressFormat.JPEG, mJpegQuality, stream);
```

Reply:
Good catch. Will update & push accordingly.